### DISCUSSION TITLE
How to insert a tab char at the beginning of each file name line like magit 1.x do in status buffer?

### DISCUSSION THREAD

**Discussion by netjune** [magit/magit#5002] [Ideas]
In magit 1.x, it shows like this:
```
Changes:
	Modified   lsp-go.el
	Modified   lsp.el
```

In 3.x, it shows like this:
```
Unstaged changes (2)
Modified   lsp-go.el
Modified   lsp.el
```
I want it to be like this:
```
Unstaged changes (2)
        Modified   lsp-go.el
        Modified   lsp.el
```

Is there customization option for this?

**Comment by tarsius** [comment]
> Is there customization option for this?

No, and I don't intend to add one.

You could implement this using an advice to `magit-insert-heading` to check the argument against a list of values that you want to prefix with some whitespace.

I don't think the indentation is *necessary* because faces can be used to make the heading and the list of files easier to tell apart.  But I can understand that one could *prefer* indentation.

However once you expand file sections to show the diffs, it would be very odd if the headers were indented, because then those headings would be **more** indented than their respective content, the diffs.

**Reply by netjune** [reply]
It is like the tree view. I think it is more intuitive. 

>However once you expand file sections to show the diffs, it would be very odd if the headers were indented, because then those headings would be more indented than their respective content, the diffs.

I think it is ok. It is like a paragraph indent in normal text.

I will try to advice to `advice to magit-insert-heading`.
Thanks.
